content/docs/04-clusters/02-data-center/01-maas/10-register-manage-maas-cloud-accounts.md

```diff
@@ -42,16 +45,21 @@ Follow these steps to create additional MAAS cloud accounts.
 3. Locate **MAAS** on the **Cloud Accounts** page and click **Add MAAS Account**.
 
 
-4. Enter values for properties listed in the following table.
+4. In the next window that displays, enter values for properties listed in the following table.
+
+    In a self-hosted environment where Palette has direct network access to MAAS, you can register a MAAS cloud account without installing the PCG. Note the **Use System Private Gateway** setting listed in the table.
```

Comment:
Add a link to the Architecture page where System CG is discussed.

Reply:
Added.